### DISCUSSION TITLE
Unable to play some webradio

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#324] [Fixed Issues] [answered]
I tried to add webradios like:
http://mega.stweb.tv/mega983/live/playlist.m3u8

or
http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/hls/uk/sbr_high/ak/bbc_radio_three.m3u8

but mpd fails to play them with error:

```
May 10 00:28:38 rAudio taskset[4635]: exception: Failed to decode http://mega.stweb.tv/mega983/live/playlist.m3u8
May 10 00:28:38 rAudio taskset[4635]: player: played "http://mega.stweb.tv/mega983/live/playlist.m3u8"
May 10 00:30:36 rAudio taskset[4635]: exception: Failed to decode http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/hls/uk/sbr_high/ak/bb>
May 10 00:30:36 rAudio taskset[4635]: player: played "http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/hls/uk/sbr_high/ak/bbc_radio_thre>

```
on same machine (raspberry pi4) playing these URLs with mpv works, I also tried playing the same URLs on a Ubuntu x86_64 machine and in an Archlinux x86_64 vm and mpd has been able to play in both.

I tried to set mpd log_level to verbose but nothing has been logged about the playing failure.
Any suggestion?

**Comment by R101G** [comment, marked as answer]
The BBC m3u8 streams play OK for me. Have you enabled ffmpeg in Player menu?

**Reply by vale-max** [reply]
....no, I hadn't. Now I enabled it and the streams play.
Thanks

**Comment by vale-max** [comment]
Solved by enabling "ffmpeg" in player settings.
